README.md

```diff
@@ -317,6 +317,24 @@ $ DEBUG=foo node -e 'var dbg = require("debug"); dbg.enable("bar"); console.log(
 => false
 ```
 
+`disable()`
+
+Will disable all namespaces. The functions returns the namespaces currently
```

Comment:
@Qix- I added some documentation here. I don’t know if this is what you wanted?

Reply:
Yes :)